src/shell/scripts/omp.ps1

```diff
@@ -286,27 +286,34 @@ Example:
     }
 
     function Update-PoshErrorCode {
-        $script:ExecutionTime = -1
         $lastHistory = Get-History -ErrorAction Ignore -Count 1
+
+        # default values
+        $script:ErrorCode = 0
```

Comment:
@JanDeDobbeleer If you move this line here, a non-zero error code cannot persist after empty hits, which breaks the original behavior.

Reply:
Ah damn it. I did validate it, but not for that use-case. Revert?

Reply:
Yep. And we have to find another way to resolve #2633.